File: 프로그래머스/unrated/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    user_id, user_pw = id_pw
    
    for db_id, db_pw in db:
        if db_id == user_id and db_pw == user_pw:
            return "login"
        elif db_id == user_id:
            return "wrong pw"
        
    return "fail"

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "";
        String id = id_pw[0];
        String pw = id_pw[1];
        
        for(int i=0;i<db.length;i++){
            if(db[i][0].equals(id)){
                if(db[i][1].equals(pw)){
                    return "login";
                }else{
                    return "wrong pw";
                }
            }
         }
        return "fail";
    }
}